### DISCUSSION TITLE
How does this work if the Venmo APIs have been retired?

### DISCUSSION THREAD

**Discussion by eric-s321** [mmohades/Venmo#81] [Q&A]
@mmohades 

Sorry if this is a naive question. I'm trying to grasp conceptually how this API works. Everything I read tells me the Venmo public APIs were discontinued in 2016. Is that the same one you are referring to [here](https://github.com/mmohades/VenmoApiDocumentation) and if so how are we still able to access it externally? I assumed since it was discontinued people would need to use the SDKs. 

I'm guessing there's something obvious I'm missing here :) but it's been nagging at me since I was playing around with your python wrapper library (which is awesome btw!) 

Any explanation would be greatly appreciated.

Thanks!
Eric

**Comment by rseabrook** [comment]
Super late reply here, but I saw this discussion and figured i could clarify.

The _**PUBLIC**_ API is retired, but all mobile apps have _**private**_ APIs that allow them to function. The data displayed in the official venmo app is requested from Venmo's servers via these private APIs. They are not intended to be used by the public, and therefore are not documented publicly. But its possible to reverse engineer them, which @mmohades has kindly done for all of us in this repo and documented his findings.
